resources/schema.json

```diff
@@ -8,6 +8,10 @@
       "description": "Name of the resource.",
       "pattern": "^[-a-zA-Z0-9]+$"
     },
+    "owner": {
+      "type": "string",
+      "description": "Optional name of the user or group who will own the resource upon creation. If blank the user creating the resource is the owner. USE WITH CAUTION: in most situations this should be left blank."
```

Comment:
This feels a little misleading since the only action available with recipes right now is clone, which totally drops the `owner` field. We'll need something like this in the future though.

Reply:
I'd rather leave it in, especially since we're already probably in need of adjusting the UI to allow for cloning into groups

Reply:
We need to make it clear that this field is totally ignored on clone.

Reply:
Okay last swing